draft-ietf-masque-h3-datagram.md

```diff
@@ -276,59 +363,88 @@ HTTP/3 datagrams enabled on this endpoint.
+
+~~~
+  +------------+----------+---------------+
+  | Frame Type |   Value  | Specification |
+  +============+==========+===============+
+  |  CAPSULE   | 0xffcab5 | This Document |
+  +------------+----------+---------------+
+~~~
+
+
 ## HTTP SETTINGS Parameter {#iana-setting}
 
 This document will request IANA to register the following entry in the
 "HTTP/3 Settings" registry:
 
 ~~~
-  +--------------+-------+---------------+---------+
-  | Setting Name | Value | Specification | Default |
-  +==============+=======+===============+=========+
-  | H3_DATAGRAM  | 0x276 | This Document |    0    |
-  +--------------+-------+---------------+---------+
+  +--------------+----------+---------------+---------+
+  | Setting Name |   Value  | Specification | Default |
+  +==============+==========+===============+=========+
+  | H3_DATAGRAM  | 0xffd276 | This Document |    0    |
+  +--------------+----------+---------------+---------+
 ~~~
 
 
-## HTTP Header Field {#iana-header}
+## Capsule Types {#iana-types}
+
+This document establishes a registry for HTTP/3 frame type codes. The "HTTP
+Capsule Types" registry governs a 62-bit space. Registrations in this registry
+MUST include the following fields:
+
+Type:
+
+A name or label for the capsule type.
+
+Value:
+
+: The value of the Capsule Type field (see {{capsule-frame}}) is a 62bit
+integer.
+
+Reference:
+
+: An optional reference to a specification for the type. This field MAY be
+empty.
+
+Registrations follow the "First Come First Served" policy (see Section 4.4 of
+{{!IANA-POLICY=RFC8126}}) where two registrations MUST NOT have the same Type.
 
-This document will request IANA to register the "Datagram-Flow-Id"
-header field in the "Permanent Message Header Field Names"
-registry maintained at
-<[](https://www.iana.org/assignments/message-headers)>.
+This registry initially contains the following entries:
 
 ~~~
-  +-------------------+----------+--------+---------------+
-  | Header Field Name | Protocol | Status |   Reference   |
-  +-------------------+----------+--------+---------------+
-  | Datagram-Flow-Id  |   http   |  std   | This document |
-  +-------------------+----------+--------+---------------+
++---------------------------+-------+---------------+
+| Capsule Type              | Value | Specification |
++---------------------------+-------+---------------+
+| REGISTER_DATAGRAM_CONTEXT | 0x00  | This Document |
++---------------------------+-------+---------------+
+| CLOSE_DATAGRAM_CONTEXT    | 0x01  | This Document |
++---------------------------+-------+---------------+
+| DATAGRAM                  | 0x02  | This Document |
++---------------------------+-------+---------------+
 ~~~
 
 
-## Flow Parameters {#iana-params}
+## Context Extension Keys {#iana-keys}
 
-This document will request IANA to create an "HTTP Datagram Flow
-Parameters" registry. Registrations in this registry MUST
+This document will request IANA to create an "HTTP Datagram Context
```

Comment:
I assume this is for the 'extension-string' parameter in Register Datagram Context.  The word key first appears here.

Reply:
These are the keys from that section, I added references.